dev-docs/explanations/translations.md

```diff
@@ -0,0 +1,66 @@
+# Translations
+
+Ubuntu Pro Client is a special package in Ubuntu: it is updated frequently in all Ubuntu LTS releases.
+That makes language packs unsuitable for providing and updating translations for Ubuntu Pro Client.
+
+In addition, Ubuntu Pro Client includes lots of language with potential commercial and contractual
+implications related to Canonical's Ubuntu Pro offering. That means we need to maintain tight control
+over the language used in all messages.
+
+## How are translations delivered and updated?
+
+Normally, packages in `main` are translated via language packs; however language packs are only updated
+in stable releases around the time of point releases. If Ubuntu Pro translations were put in the language pack,
+then the language pack wouldn’t get updated as we continue to SRU new Ubuntu Pro Client versions back to
+old Ubuntu releases. That would cause the translations provided by the language pack to drift out of sync with
+the version of Ubuntu Pro Client installed.
+
+Because of this, Ubuntu Pro Client is not included in the Ubuntu language pack and instead ships all translations
+directly in our own source package. However, translations are kept in a separate binary package so that it is not
+required to be installed with Ubuntu Pro Client (and therefore not required in `ubuntu-minimal`). Translations are
+in a binary package named: `ubuntu-pro-client-l10n`.
+The `ubuntu-advantage-tools` binary package _Recommends_ `ubuntu-pro-client-l10n`, but doesn't require it.
+
+In order to ship translation files (`.mo` files) in the `ubuntu-pro-client-l10n` binary package, this package is
+explicitly excluded from language pack translation stripping done by the `pkgbinarymangler` package
+(see [LP: #2037584](https://bugs.launchpad.net/ubuntu/+source/pkgbinarymangler/+bug/2037584)).
+
+## How are translations maintained and added?
+
+Translations live as `.po` files in the `debian/po/` directory.
+
+Because many of the pro-client’s messages are commercial in nature, a bad translation could potentially have odd or
+misleading contractual implications. Because of that, we generally want all translations to come from Canonical employees.
+
+Regarding contributions, we will:
+- Accept Canonical-employee PRs of translations in Github.
+- Consider community bugs/issues regarding translation wording/suggestions
+  - We’ll get input from a Canonical employee who speaks the language in question.
+- **Not** accept community PRs directly changing translations
+
+We generally will not support translations to a language that no Canonical-employee understands and is willing to review/help maintain.
```

Comment:
Wording is a bit confusing, I had to read it twice. Suggest something more like:

For a language to be supported, there must be at least one Canonical employee who speaks/reads/understands that language and is willing to review/help maintain the translation.